.gometalinter.json

```diff
@@ -0,0 +1,27 @@
+{
+	"DisableAll": true,
+	"Enable": [
+		"deadcode",
+		"dupl",
+		"errcheck",
+		"gas",
```

Comment:
`gas` is somewhat nasty, too.
It can stay for now.

Reply:
Ok.